packages/react/src/runtimes/remote-thread-list/RemoteThreadListThreadListRuntimeCore.tsx

```
@@ -317,7 +339,7 @@
       optimistic: (state) => {
         return updateStatusReducer(state, threadId, "regular");
       },
-      then: (state, { remoteId }) => {
+      then: (state, { remoteId, externalId }) => {
```

Biome (1.9.4)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)